### ISSUE TITLE
YOLOv4-CSP-X and YOLOR-CSP-X

### ISSUE THREAD

**Issue by kadmor** [WongKinYiu/yolor#145]
Hello, thanks for the great repository and paper. Please let me know:
What is the '....- X' in the model name? is this a reference to [YOLOX](https://arxiv.org/abs/2107.08430)?
(p.s. I mean YOLOv4-CSP-X, YOLOR-CSP-X)

**Comment by WongKinYiu** [comment]
No, in YOLOv4-CSP-S, YOLOv4-CSP(-L), and YOLOv4-CSP-X, -S and -X apply model scaling method used in [ultralytics](https://github.com/ultralytics/yolov5/tree/c14368d768749734e1a017ed8e750b33597e8746).

**Comment by kadmor** [comment]
> No, in YOLOv4-CSP-S, YOLOv4-CSP(-L), and YOLOv4-CSP-X, -S and -X apply model scaling method used in [ultralytics](https://github.com/ultralytics/yolov5/tree/c14368d768749734e1a017ed8e750b33597e8746).

Thanks a bunch.
